### DISCUSSION TITLE
Does deepmd have support ROCM GPU (MI series chip) version?

### DISCUSSION THREAD

**Discussion by wangych6** [deepmodeling/deepmd-kit#1999] [Q&A] [answered]
Does deepmd have **support ROCM GPU** (MI series chip) version， and how should I install the version that supports AMD's GPU？（I have tried：  docker pull deepmodeling/dpmdkit-rocm:dp2.0.3-rocm4.5.2-tf2.6-lmp29Sep2021，but not work）.

**Comment by njzjz** [comment, marked as answer]
You must confirm the lowest version of the ROCm toolkit that supports your hardware, and use the compatible toolkit to build deepmd-kit.

**Comment by wanghan-iapcm** [comment]
Please provide a full log of the error messages.
